praktikumsplaner-frontend/apigateway/src/main/java/de/muenchen/oss/praktikumsplaner/configuration/GuiConfiguration.java

```diff
@@ -27,7 +27,7 @@
 public class GuiConfiguration {
 
     /**
-     * A router which returns the index.html as a resource.
+     * A index which returns the index.html as a resource.
```

Comment:
```suggestion
     * A router which returns the index.html as a resource.
```
why was this changed?